provisioning/templates-api/kubernetes/templates/api/deployment.yaml

```diff
@@ -55,26 +55,30 @@ spec:
             # 80% of the resources.limits.memory
             - name: GOMEMLIMIT
               value: "550MiB"
-            - name: TEMPLATES_API_STORAGE_API_HOST
+            - name: TEMPLATES_NODE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
```

Comment:
Pridane, tj. pod hostname.

Ostatne zmeny su len `TEMPLATES_API` -> `TEMPLATES_`.

Ze som to nerozbil kontroluje test pomocou Minikube, kde sa to cele nasadi a vyskusa, vid PR.